packages/account/src/Components/Routes/binary-routes.jsx

```diff
@@ -19,8 +19,8 @@ const BinaryRoutes = props => {
             }}
         >
             <Switch>
-                {getRoutesConfig({ is_appstore }, is_social_signup).map((route, idx) => (
-                    <RouteWithSubRoutes key={idx} {...route} {...props} />
+                {getRoutesConfig({ is_appstore }, is_social_signup).map(route => (
+                    <RouteWithSubRoutes key={`${route?.getTitle?.()}${route?.path}`} {...route} {...props} />
```

Comment:
Do we need optional chain here?
what if the data was null? we don't have key on that occasion

Reply:
Hello @niloo-fs,

one route doesn't have a `getTitle()` defined on it, which is the default / home route. so it returns undefined and the  `route.path` returns an empty string, because it's the default / home route. so it's key in the virtual dom will be the string "undefined" while others will have their respective values. Optional chaining is just a precaution and not really necessary